src/model/ident_env.cppo.ml

```diff
@@ -165,6 +176,8 @@ let extract_signature_tree_item item =
             | Some id -> `Module (id, false) :: items
             | None -> items)
           mds []
+    | Tsig_value {val_id; _} ->
+        [`Value (val_id, false)]
```

Comment:
This is in the `#if OCAML_MAJOR = 4 && OCAML_MINOR >= 10` branch, this will be removed on lower versions and the pattern will not be exhaustive.
I think you can move it to after the `#endif`.